x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -445,6 +448,9 @@ export interface EndpointEvent {
       entity_id: string;
       name?: string;
     };
+    Ext: {
```

Comment:
We use a capital here to avoid collisions with ecs core since the ancestry array is a custom extension field.

Reply:
I was wondering what `Ext` was for, thanks for the clarification